Web/starter_project_iconics/components/common/blogCard.tsx

```diff
@@ -0,0 +1,165 @@
+            {(() => {
+              if (feature === 'likes') {
+                let val: string = ''
+                for (const [value, abbreviation] of abbreviations) {
+                  if (card.likes >= value) {
+                    const roundedNum = (card.likes / value).toFixed(1)
+                    val = `${roundedNum}${abbreviation} likes`
+                  }
+                }
+                return (
+                  <span className="flex items-center">
+                    <span
+                      className="card_writter_likes m-0 text-sm font-medium text-gray-500"
+                      data-likes={card.likes}
+                      style={{ display: 'inline-block', marginRight: '14px' }}
+                    >
+                      <MdOutlineModeComment size={15} />
+                    </span>
+                    <span
+                      className="card_writter_likes m-0 text-sm font-medium text-gray-500"
+                      data-likes={card.likes}
+                      style={{ display: 'inline-block' }}
+                    >
+                      {val}
+                    </span>
+                  </span>
+                )
+              } else {
+                return (
+                  <span className="flex items-center">
+                    {
+                      <span
+                        className={
+                          card.status === 'approved'
+                            ? 'text-green-600 '
+                            : card.status === 'declined'
+                            ? 'text-red-600 '
+                            : 'text-orange-500 '
+                        }
+                        style={{
+                          display: 'inline-block',
+                          marginRight: '10px',
+                        }}
+                      >
+                        {statusIcon[card.status || 'pending']}
+                      </span>
+                    }
+                    <span
+                      className={
+                        card.status === 'approved'
+                          ? 'text-green-600 text-sm'
+                          : card.status === 'declined'
+                          ? 'text-red-600 text-sm'
+                          : 'text-orange-500 text-sm'
+                      }
+                      style={{ display: 'inline-block' }}
+                    >
+                      {card.status.toUpperCase()}
+                    </span>
+                  </span>
+                )
+              }
+            })()}
+
+            <button
+              className="card_read_more px-4 py-2 text-sm rounded-md border-none bg-white text-purple-500 font-bold cursor-pointer transition duration-200 ease-in-out"
+              onClick={handleReadMore}
+            >
+              Read More
+            </button>
+          </div>
+        </div>
+      </div>
+    </div>
+  )
+}
```

Comment:
Is there a way to make this page smaller, too many lines. Maybe you could import some of the jsx you return based on the condition